trino-rest-github/src/main/java/pl/net/was/rest/github/Sync.java

```diff
@@ -74,7 +75,8 @@ public static void main(String[] args)
                 "TRINO_PASSWORD",
                 "EMPTY_INSERT_LIMIT",
                 "CHECK_STEPS_DUPLICATES",
-                "CHECK_ARTIFACTS_DUPLICATES");
+                "CHECK_ARTIFACTS_DUPLICATES",
+                "QUERY_RETRY_DELAY_MS");
```

Comment:
It should be just query delay, that is sleep for this much - query run time if it's > 0. We should not expect to hit the secondary limit, rather try to avoid it.

Reply:
Renamed variable to `QUERY_DELAY_MS`